alinka/widget/containers/main_body/content/settings/settings_tabs/team_member_tab/__init__.py

```diff
@@ -10,3 +10,11 @@ def __init__(self, parent: QWidget):
         layout = QVBoxLayout(self)
         self.team_members_table_group = TeamMemberTableGroup(self)
         layout.addWidget(self.team_members_table_group)
+
+    @property
+    def is_valid(self) -> bool:
```

Comment:
I am not sure if I get this concept as well.

Reply:
This property should return if given container is valid or not. According to the ticket I implemented it only for support center settings, and left it as a stub on the rest of containers for next iterations. Not sure if I addressed correctly your questions. If you can elaborate it a bit?

Reply:
Maybe we can add a comment there, that these are stubs and should be implemented in the future (in specific ticket).

Reply:
Comment, and reference to ticket have been added.